scipy/spatial/distance.py

```diff
@@ -1256,16 +1259,20 @@ def jensenshannon(p, q, base=None):
     0.46450140402245893
     >>> distance.jensenshannon([1.0, 0.0, 0.0], [1.0, 0.0, 0.0])
     0.0
+    >>> distance.jensenshannon([[1.0, 2.0]], [[0.5, 1.5]], axis=0)
```

Comment:
Maybe having a larger example would be more explanatory. I was puzzled at first with this first example.